src/Sql/AbstractSql.php

```diff
@@ -241,7 +241,7 @@ protected function createSqlFromSpecificationAndParameters($specifications, $par
             if (isset($paramSpecs[$position]['combinedby'])) {
                 $multiParamValues = [];
                 foreach ($paramsForPosition as $multiParamsForPosition) {
-                    $ppCount = count($multiParamsForPosition);
+                    $ppCount = @count($multiParamsForPosition);
```

Comment:
Our coding guidelines have a preference for **NOT** using error suppression. In this case, I'd do one of the following:

- See if `$multiParamsForPosition` is an array, and, if so, return `count($multiParamsForPosition)`
- If `$multiParamsForPosition` is an object, test if it is `Countable`, and, if so, return `$multiParamsForPosition->count()`
- Otherwise, raise an exception (invalid data is present!)